bpf/lib/maps.h

```diff
@@ -192,6 +192,21 @@ struct {
 	__uint(map_flags, BPF_F_NO_PREALLOC);
 } NODE_MAP __section_maps_btf;
 
+struct node_value {
+	__u16 id;
+	__u8  spi;
+	__u8  pad;
```

Comment:
I jumped straight here to ask for padding and it's already here :heart:

(We probably will need this padding at some point to have a flag that indicates the type of node IP, to help with masquerading. Best to migrate once now :sweat_smile: )